v2x_ros_driver/launch/v2x_ros_driver.launch.py

```diff
@@ -42,7 +82,11 @@ def generate_launch_description():
 
     configuration_delay = LaunchConfiguration('configuration_delay')
     declare_configuration_delay_arg = DeclareLaunchArgument(
-        name ='configuration_delay', default_value='4.0')
+        name ='configuration_delay', default_value='2.0')
```

Comment:
Probably would be best to add a 'description' string to each of these launch arguments to help external users quickly recognize what they are for. Same applies for `enable_v2x_lifecycle` as well.